### PR TITLE
added footer

### PR DESCRIPTION
**Pull request by srishtigautam21**
added footer to my CL
did some typography changes 
and small refactors

### CODE REVIEW

File: component/Avatar/avatar.css [hunk 1]
```diff
@@ -22,30 +22,30 @@
...
     margin:4.5rem 2rem 0rem 0rem;
 }
 .xs-text-avatar{
     background-color:var(--COLOR-PRIMARY);
     font-size: 1.3rem;
-    font-weight: 600;
+    font-weight: 500;
     text-align: center;
     padding-top:6px;
     margin:5.5rem 0rem 0rem 0rem;
```
Comment (zoef-yt) [inline component/Avatar/avatar.css]: units are not required if the value is 0 so rem can be removed here

File: css/components.css [hunk 2]
```diff
@@ -1,4 +1,5 @@
-@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap');
+/* @import url('https://fonts.googleapis.com/css2?family=Roboto:wght@300&display=swap'); */
```
Comment (zoef-yt) [inline css/components.css]: remove commented code

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by Omkar-Ghate — APPROVED** [review]
Great work. Just confirm for the footer in the images component.

**Review by zoef-yt — APPROVED** [review]
Approved

**Pull request merged**